JunitHomework/src/test/java/ro/sciit/junit/tests/NumberReplacerTest.java

```diff
@@ -0,0 +1,101 @@
+package ro.sciit.junit.tests;
+
+import org.junit.Before;
+import org.junit.Test;
+import ro.sciit.junit.homework.Utility;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class NumberReplacerTest {
```

Comment:
If you are testing the utilityClass why are you giving different class naming here?

Please create an UtilityClassTest and move all the methods from both test classes to the UtilityClassTest.
Thx